Baymax/Plist Viewer/PlistValue.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  Utilities.swift
+//  Baymax
+//
+//  Created by Matthew Cheetham on 13/11/2018.
+//  Copyright © 2018 3 SIDED CUBE. All rights reserved.
+//
+
+import Foundation
+
+/// A protocol we will use to conform classes which can be represented in an Info.plist to
+protocol PlistValue {
+    
+    func toString() -> String
+}
+
+func plistValueFrom(_ value: Any) -> PlistValue? {
```

Comment:
I guess except from `Dictionary` and `Array` (because of the `compactMap`) this is simply `return value as? PlistValue`?
So could do them first then return this. Unless you want to be explicit with your types then you can keep this way :D

Reply:
Can't quite do that unfortunately, because the extensions to conform are on the `Swift` string, but that doesn't effectively conform `NSString` or any of it's weird subclasses to `PlistValue` too!